### DISCUSSION TITLE
Connected to server but no internet/ cannot ping the remote network

### DISCUSSION THREAD

**Discussion by jhay06** [strongswan/strongswan#1088] [Q&A]
Hi , 
I managed to work strongswan configuration to connect to our company network 
but when i try to ping 1 of the internal  IP of the company , it doesn't ping .
please see below result of ipsec statusall
![image](https://user-images.githubusercontent.com/5191437/173187324-76f1cdf4-324a-4225-accb-c37f8a5db874.png)

and per checking on the ip xfrm there's no policy installed 
![image](https://user-images.githubusercontent.com/5191437/173187351-e95be81b-58b0-4c76-875c-db56d60d3a0e.png)

please see below config i used 

conn Forcepoint
	authby=rsasig
	auto=start
	left=%defaultroute
	leftid=me@InternalDomain
	leftcert=me.crt
	leftsendcert=always
	right=company.ip
	rightid=%company.ip
	leftsubnet=192.168.XXX.XXX/24
	rightsubnet=192.168.XXX.XXX/24
	compress=no
	keyexchange=ikev1
	ike=3des-sha1-modp1024
	esp=3des-sha1-modp1024
	modeconfig=pull
	rekey=yes
	type=tunnel
	mobike=no

Can someone help me with this ? 

I tried the same config on libreswan and i able to connect and ping using the libreswan 

i

**Comment by tobiasbrunner** [comment]
That output is not useful and incomplete (see #196). But you definitely don't have matching policies installed (the ones you see are the socket policies to prevent IKE traffic from getting tunneled). So I guess the Quick Mode exchange failed for some reason (check the log).
